dotCMS/src/main/webapp/html/js/dotcms/dijit/FileBrowserDialog.jsp

```diff
@@ -24,10 +24,15 @@
                             </div>
                             <div class="portlet-toolbar__actions-secondary">
                                 <div class="file-selector-tree__views-menu">
-                                    <span><%= LanguageUtil.get(pageContext, "View")%>:</span>
-                                    <img dojoAttachPoint="listViewIcon" src="/html/images/icons/application-detail.png" alt='<%= LanguageUtil.get(pageContext, "list-view")%>' title='<%= LanguageUtil.get(pageContext, "list-view")%>'>
-                                    <img dojoAttachPoint="detailViewIcon" src="/html/images/icons/application-list.png" alt='<%= LanguageUtil.get(pageContext, "details-view")%>' title='<%= LanguageUtil.get(pageContext, "details-view")%>'>
-                                    <img dojoAttachPoint="thumbnailViewIcon" src="/html/images/icons/application-icon.png" alt='<%= LanguageUtil.get(pageContext, "thumbnails-view")%>' title='<%= LanguageUtil.get(pageContext, "thumbnails-view")%>'>
+                                    <div style="margin:10px;"><%= LanguageUtil.get(pageContext, "View")%>:</div>
+                                    <div dojoAttachPoint="detailViewIcon" style="margin:10px;cursor:pointer;">
+                                        <i  class="material-icons">format_list_bulleted</i>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5337824)